include/asterisk/channel.h

```diff
@@ -4446,6 +4446,23 @@ struct ast_str *ast_channel_dialed_causes_channels(const struct ast_channel *cha
  */
 struct ast_control_pvt_cause_code *ast_channel_dialed_causes_find(const struct ast_channel *chan, const char *chan_name);
 
+/*!
+ * \brief Retrieve a ref-counted cause code information structure iterator
+ *
```

Comment:
\since version should be added

Reply:
We're currently in release candidates, so it would be the next major version after these. 20.17.0, 22.8.0, 23.1.0